frontend/src/react/Exam/CreateExam/Exercice/ExerciceList/ExerciceListDisplayer.js

```diff
@@ -37,14 +36,14 @@ export default class ExerciceListDisplayer extends React.Component {
                 </div>
                 <SortableContainer onSortEnd={this.props.onSortEnd}>
                     {typeof this.props.exercices !== 'undefined' && this.props.exercices.map((value, idx) => this.props.isExtended ?
-                        <Exercice key={uniqid()} id={this.props.id} exercices={value}
+                        <Exercice key={this.props.id} id={this.props.id} exercices={value}
```

Comment:
Why not using uniquid in this case ?

Reply:
Because uniqid will return a different id each time we render, which is exatly not what we want. The key prop is used to tell react which component correspond to which js object, so we don't want it to change every time it is re-rendered

Reply:
But anyway, it doesn't work well for now because the whole branch is a mess; I'm trying to fix it.